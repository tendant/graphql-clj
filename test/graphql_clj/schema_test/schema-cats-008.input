type Hello {
  world(flag: Boolean): String
}
