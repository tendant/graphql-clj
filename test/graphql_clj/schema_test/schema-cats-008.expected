{:objects
 {:Hello
  {:fields {:world {:args {:flag {:type Boolean}}, :type String}}}}}
